scripts/migrate_metadata.py

```
@@ -27,17 +27,29 @@
+    errors: int
+    target_version: str
+
+
+if TYPE_CHECKING:
     MigrationFunc: TypeAlias = Callable[[dict, Path | None], tuple[dict, list[str]]]
 
 # プロジェクトルートをパスに追加
 sys.path.insert(0, str(Path(__file__).parent.parent))
 
-from src.managers.storage_manager import METADATA_VERSION
+from src.managers.storage_manager import METADATA_VERSION  # noqa: E402
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)